app/assets/javascripts/darkswarm/controllers/products_controller.js.coffee

```diff
@@ -4,15 +4,18 @@ angular.module('Darkswarm').controller "ProductsCtrl", ($scope, $sce, $filter, $
   $scope.query = ""
   $scope.taxonSelectors = FilterSelectorsService.createSelectors()
   $scope.propertySelectors = FilterSelectorsService.createSelectors()
+  $scope.producerPropertySelectors = FilterSelectorsService.createSelectors()
```

Comment:
Do we need to add more detail for testing the supplier property filtering vs product property filtering?

Reply:
Done!